tests/unit/metadata_manager_test.py

```diff
@@ -177,82 +179,62 @@ def test_extract_meta_data(metadata_manager: MetadataManager):
 """
 
 
-def test_start(mocker, metadata_manager: MetadataManager):
+def test_start(metadata_manager: MetadataManager):
```

Comment:
This test now actually tests the whole extractor execution inclusive fetching data from splash and lighthouse (where the requests are mocked away so the tests runs without the docker containers).

Reply:
Aeh, mhh yeah, I might want to rename the `extractors` to `whitelist`? Its an optional list, if not provided, all extractors will be used.

Reply:
renamed!